sdcm/tester.py

```diff
@@ -21,6 +21,7 @@
 import subprocess
 import datetime
 import platform
+from textwrap import dedent
```

Comment:
We need to add this package to our requirements.

Reply:
it is in standard Python module library :)